clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -207,6 +174,46 @@ Sema::BuildSYCLIntelFPGAIVDepAttr(const AttributeCommonInfo &CI, Expr *Expr1,
       SYCLIntelFPGAIVDepAttr(Context, CI, SafeLenExpr, ArrayExpr, SafelenValue);
 }
 
+template <typename FPGALoopAttrT>
+FPGALoopAttrT *Sema::BuildSYCLIntelFPGALoopAttr(const AttributeCommonInfo &A,
+                                                Expr *E) {
+
```

Comment:
Suggest documenting that `intelfpga::ivdep` is handled by a separate function - after all, it's also an "FPGALoopAttrT"

Reply:
I don't think it's really needed